src/table/TableWrapper.jsx

```diff
@@ -51,6 +62,33 @@ export default function TableWrapper(props) {
     return null;
   }
 
+  useEffect(() => {
+    window.addEventListener('keydown', (event) =>
+      updatePage(
+        event,
+        localState.current.totalRowSize,
+        localState.current.page,
+        localState.current.rowsPerPage,
+        setPageInfo,
+        setPage
+      )
+    );
+    window.addEventListener('keyup', handleKeyUp);
```

Comment:
If you look at the keyDown event there are booleans for ctrl, meta and shift key. they are true when they are held down and pressing an other key. So no need for two listeners here.

Reply:
the keyup  is used to delete each element once their associated key has been released
```
const handleKeyUp = (event) => {
  delete keysPressed[event.key];
};
```

Reply:
yeah but you don't have to store which keys are pressed. all that info is on the keydown event, e.g. https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/ctrlKey

Reply:
Nice. Now I know that and did an update. Thanks.